lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -528,8 +515,8 @@ export class UserAgentApplication {
 
         // Track the acquireToken progress
         this.cacheStorage.setItem(TemporaryCacheKeys.INTERACTION_STATUS, Constants.inProgress);
-        const scope = request.scopes ? request.scopes.join(" ").toLowerCase() : this.clientId.toLowerCase();
-        this.logger.verbosePii(`Serialized scopes: ${scope}`);
+        const requestSignature = request.scopes ? request.scopes.join(" ").toLowerCase() : this.clientId.toLowerCase();
```

Comment:
Should the else condition be `Constants.oidcScopes`?

Reply:
Given this is only used as the request signature I didn't see an absolute need to change it. Mostly because I didn't know the reasoning behind originally using clientId (does it need to match the scopes for any specific reason?), so I didn't want to change this without knowing.

Reply:
Since we moved the translation to happen before this I don't think it's wise to use clientId as a single scope or default. Even if it's not used for anything of substance today it may be used in the future when we may forget about this.

Reply:
I did change the variable name for the same reason, to avoid future confusion. If there's no reason too keep the clientId as request signature, I'll change it to use OIDC scopes as default signature itself. Thanks!